src/keri/app/agenting.py

```diff
@@ -74,10 +74,13 @@ def receipt(self, pre, sn=None):
         clients = dict()
         doers = []
         for wit in wits:
-            client, clientDoer = httpClient(hab, wit)
-            clients[wit] = client
-            doers.append(clientDoer)
-            self.extend([clientDoer])
+            try:
+                client, clientDoer = httpClient(hab, wit)
+                clients[wit] = client
+                doers.append(clientDoer)
+                self.extend([clientDoer])
+            except Exception as e:
```

Comment:
Can we catch something more specific here?

Reply:
yes, but I need to import `socket`. Is that ok.

Reply:
I confined it to  two specific exceptions